config/game_options.txt

```diff
@@ -563,3 +563,6 @@ RATCAP 64
 
 ## Maximum fine for a citation
 MAXFINE 2000
+
+## Uncomment to enable SDQL spells
```

Comment:
```suggestion
## Uncomment to enable SDQL spells
## Warning: SDQL is a powerful tool and can break many things or expose security sensitive information.
## Giving players access to it has major security concerns, be careful and deliberate when using this feature.
```

Reply:
The players don't choose what SDQL query to execute with the spells they are given - the admin defines the query beforehand. That said, a poorly written query could indeed cause the problems you describe in your suggestion.

Reply:
Yeah I'm aware, just wanted to give people context here about what it is they're enabling